core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/WorkspaceMigration.scala

```diff
@@ -54,145 +56,151 @@ final case class WorkspaceMigration(id: Long,
+    val tmpBucket = MigrationColumnName("TMP_BUCKET")
+    val tmpBucketCreated = MigrationColumnName("TMP_BUCKET_CREATED")
+    val workspaceBucketTransferJobIssued = MigrationColumnName("WORKSPACE_BUCKET_TRANSFER_JOB_ISSUED")
+    val workspaceBucketTransferred = MigrationColumnName("WORKSPACE_BUCKET_TRANSFERRED")
+    val workspaceBucketDeleted = MigrationColumnName("WORKSPACE_BUCKET_DELETED")
+    val finalBucketCreated = MigrationColumnName("FINAL_BUCKET_CREATED")
+    val tmpBucketTransferJobIssued = MigrationColumnName("TMP_BUCKET_TRANSFER_JOB_ISSUED")
+    val tmpBucketTransferred = MigrationColumnName("TMP_BUCKET_TRANSFERRED")
+    val tmpBucketDeleted = MigrationColumnName("TMP_BUCKET_DELETED")
+    val requesterPaysEnabled = MigrationColumnName("REQUESTER_PAYS_ENABLED")
+    val unlockOnCompletion = MigrationColumnName("UNLOCK_ON_COMPLETION")
+    val workspaceBucketIamRemoved = MigrationColumnName("WORKSPACE_BUCKET_IAM_REMOVED")
+
+    val allColumns = s"""$id,
+                        |$workspaceId,
+                        |$created,
+                        |$started,
+                        |$updated,
+                        |$finished,
+                        |$outcome,
+                        |$message,
+                        |$newGoogleProjectId,
+                        |$newGoogleProjectNumber,
+                        |$newGoogleProjectConfigured,
+                        |$tmpBucket,
+                        |$tmpBucketCreated,
+                        |$workspaceBucketTransferJobIssued,
+                        |$workspaceBucketTransferred,
+                        |$workspaceBucketDeleted,
+                        |$finalBucketCreated,
+                        |$tmpBucketTransferJobIssued,
+                        |$tmpBucketTransferred,
+                        |$tmpBucketDeleted,
+                        |$requesterPaysEnabled,
+                        |$unlockOnCompletion,
+                        |$workspaceBucketIamRemoved""".stripMargin
+
+    implicit val getGoogleProjectId = GetResult(r => Option(r.nextString).map(GoogleProjectId))
+    implicit val getGoogleProjectNumber = GetResult(r => Option(r.nextString).map(GoogleProjectNumber))
+    implicit val getGcsBucketName = GetResult(r => Option(r.nextString).map(GcsBucketName))
+    implicit val getWorkspaceMigration = GetResult(r => WorkspaceMigration(r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, unsafeFromEither(Outcome.fromFields(r.<<, r.<<)), r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<))
+
+    object SetAnyParameter extends SetParameter[Any] {
+      def apply(v: Any, pp: PositionedParameters) {
+        v match {
+          case n: Long => pp.setLong(n)
+          case s: String => pp.setString(s)
+          case t: Timestamp => pp.setTimestamp(t)
+          case vo: ValueObject => pp.setString(vo.value)
+          case b: Boolean => pp.setBoolean(b)
+          case Some(opt: Any) => SetAnyParameter(opt, pp)
```

Comment:
Why not a case for `None`?

Reply:
because it doesn't happen. It is also difficult because `pp.setNull` requires a sql type and that info has been lost